inspektr/src/main/java/org/ccci/idm/user/inspektr/resolver/ReturnUserAuditResourceResolver.java

```diff
@@ -0,0 +1,24 @@
+package org.ccci.idm.user.inspektr.resolver;
+
+import static org.ccci.idm.user.inspektr.util.InspektrSerializationUtils.userToStringHelper;
+
+import org.aspectj.lang.JoinPoint;
+import org.ccci.idm.user.User;
+import org.jasig.inspektr.audit.spi.AuditResourceResolver;
+
+public class ReturnUserAuditResourceResolver implements AuditResourceResolver {
```

Comment:
I'm not quite familiar with how these are used. Are they used by the inspektr framework in conjunction with the `@Audit` annotation as in
https://github.com/CruGlobal/thekey/blob/master/support-selfservice/src/main/java/me/thekey/cas/selfservice/api/SelfServiceApiControllerImpl.java#L110  ?